packages/terra-form-input/src/Input.module.scss

```diff
@@ -1,10 +1,10 @@
 :local {
   .form-input {
     background-color: var(--terra-form-input-background-color, #fff);
-    background-image: var(--terra-form-input-focus-animation-background);
+    background-image: var(--terra-form-input-focus-animation-background-image);
     background-position: 0 100%;
     background-repeat: no-repeat;
-    background-size: var(--terra-form-input-focus-background-start);
+    background-size: var(--terra-form-input-focus-start-background-size);
```

Comment:
nit: I know we are using background-size for focus animation, and this variable name help remember our use of it, though I think we can drop the `focus-start` part from this variable name mainly to be consistent with our other variable names.

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/d89bf406444dc93fa1237dae7fc382f0ec92b6b0#diff-462afbdb742ea35bac7504edbfd87d7dR4